### DISCUSSION TITLE
I can't not download from bilibili.

### DISCUSSION THREAD

**Discussion by adsm13** [aclap-dev/video-downloadhelper#1228] [Help and Q&A] [answered]
### Your question

Is someone else having this problem?
this video: https://www.bilibili.com/video/BV1KuvTeCEiR/?t=1.4757&spm_id_from=333.1350.jump_directly

### Addon & CoApp version and details

version: 9.1.0.43
target: microsoft
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files (x86)\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":""}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 Edg/128.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "inbrowser",
    "download_directory": "C:\\Users\\adsma\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\adsma\\Downloads"
        }
    },
    "concurrent_downloads_max": 2,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": true,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 1,
    "successfull_dl": 30
}

**Comment by Mattmjs** [comment, marked as answer]
I think this site operates in a similar way to #703 , are bilibili.com & bilibili.tv related ?
I could open the video and audio from bilibili.tv in separate tabs, on bilibili.com I can only open the audio. It says file not found when
attempting to open the video.

My only solution is to change the site to it's mobile view and there are two ways do do that.

Open the emulation mode in Edge, first by pressing **Ctrl+Shift+I**  then **Ctrl+Shift+M** or look for the button to toggle it.
Then choose a mobile device.

Add an extension to change the user agent, after that choose a browser on android or ios.

**Reply by GrampaWildWilly** [reply]
Confirmed.  Switching to an Android user_agent allowed VDH to download this no problem.  By the way, the URL should be just this:

https://www.bilibili.com/video/BV1KuvTeCEiR/?t=1.4757

The rest of the URL shown in the original post is unnecessary information & not needed to visit the web page.

**Reply by adsm13** [reply]
Thank you guys, it totally worked.
Answering your question, yup, bilibili.com & bilibili.tv are related.
